Drive/rover_drive_system.hpp

```diff
@@ -110,13 +108,48 @@ class RoverDriveSystem
     back_wheel_.SetHubSpeed(speed);
   };
 
-  /// Updates Mission Control /drive/status endpoint with rover's current status
-  void SendGETRequest();
+  /// Send HTTP GET request and updates /drive/status endpoint with rover's
+  /// telemetry readings
+  /// @return true if GET request is 200
+  bool GETRequest()
+  {
+    GETRequestParameterConstructor();
+    // TODO: Verify GET param is correct
+    // TODO: Need to implement Esp class & construct params for GET request
+    // EX: &mission_control_data_.response_body =
+    // esp_.GETDrive(mission_control_data_.request_parameter);
+    bool successful_request = true;
+    return successful_request;
+  };
+
+  /// Constructs parameters for GET request
+  void GETRequestParameterConstructor()
+  {
+    // TODO: Add state of charge for battery & is there a clean way to do this?
+    mission_control_data_.request_parameter =
+        "drive?is_operational=" +
+        std::to_string(mission_control_data_.is_operational);
+    mission_control_data_.request_parameter +=
+        "&drive_mode=" + std::to_string(static_cast<char>(current_mode_));
+    mission_control_data_.request_parameter += "&battery=" + std::to_string(50);
+    mission_control_data_.request_parameter +=
+        "&left_wheel_speed=" + std::to_string(left_wheel_.GetSpeed());
+    mission_control_data_.request_parameter +=
+        "&left_wheel_angle=" + std::to_string(left_wheel_.GetPosition());
+    mission_control_data_.request_parameter +=
+        "&right_wheel_speed=" + std::to_string(right_wheel_.GetSpeed());
+    mission_control_data_.request_parameter +=
+        "&right_wheel_angle=" + std::to_string(right_wheel_.GetPosition());
+    mission_control_data_.request_parameter +=
+        "&back_wheel_speed=" + std::to_string(back_wheel_.GetSpeed());
+    mission_control_data_.request_parameter +=
+        "&back_wheel_angle=" + std::to_string(back_wheel_.GetPosition());
```

Comment:
You can use `snprintf()` rather than this. I feel like having a format string and a list of parameters would be easier to read than this.